### DISCUSSION TITLE
I removed the whitespace from the icon. Didn't think it was worth a PR, but thought I'd share it.

### DISCUSSION THREAD

**Discussion by LuckyMcBeast** [warmuuh/milkman#202] [Show and tell]
I just downloaded the Linux source of Milkman 5.9.0 and noticed that the icon has does not have transparency in the background. I removed the unsightly white space from the icon and thought I'd share it here. I originally tried to use a PNG, but for whatever reason, the transparency still wasn't being picked up, so exported it as a Plain SVG file instead and it works like a charm, at least on Gnome 45 (Nobara 39, which derives from Fedora).

PNG File (Didn't work for .desktop icon):
![milk-bottle-trans](https://github.com/warmuuh/milkman/assets/19576746/4eca229a-0aa3-44e2-bf88-d901370a2c61)

SVG File (Worked for .desktop icon):
![milk-bottle-trans](https://github.com/warmuuh/milkman/assets/19576746/2c591f6d-5166-476f-969e-27867285d0b2)

Screenshots:
![Screenshot from 2024-06-15 15-14-38](https://github.com/warmuuh/milkman/assets/19576746/8fd65394-4728-4ad0-80d3-442099e93712)
![Screenshot from 2024-06-15 15-17-42](https://github.com/warmuuh/milkman/assets/19576746/f9263b2b-061c-4674-b857-64dcbced18f1)

**Comment by warmuuh** [comment]
just to be sure ( i dont have linux, so i cant try):
did you change Milkman.desktop file for this to 
```patch
[Desktop Entry]
Name=Milkman
Exec=$BASE_DIR/milkman.sh
- Icon=$BASE_DIR/milk-bottle.png
+ Icon=$BASE_DIR/milk-bottle.svg
Terminal=false
Type=Application
Keywords=milkman;postman;rest;http;sql;jdbc;java;
```

i read different things about how to specify this file, i.e. with extension or without extension ...

**Reply by LuckyMcBeast** [reply]
That's exactly what I did, yes. From my ~/.local/share/applications/Milkman.desktop:
```
[Desktop Entry]
Name=Milkman
Exec=/home/luckymcbeast/Sources/milkman-5.9.0/milkman.sh
Icon=/home/luckymcbeast/Sources/milkman-5.9.0/milk-bottle-trans.svg
Terminal=false
Type=Application
Keywords=milkman;postman;rest;http;sql;jdbc;java;
```
